tests/integrations/raftstore/test_conf_change.rs

```diff
@@ -863,3 +863,34 @@ fn test_conf_change_fast() {
     must_get_equal(&cluster.get_engine(2), b"k1", b"v1");
     assert!(timer.saturating_elapsed() < Duration::from_secs(5));
 }
+
+#[test_case(test_raftstore::new_node_cluster)]
+#[test_case(test_raftstore::new_server_cluster)]
```

Comment:
```suggestion
#[test_case(test_raftstore_v2::new_node_cluster)]
```

Reply:
No need to test server cluster, the test case is not related to the transport layer.